samples/logging/mslogging-custom/MsLogging_1/Sample/Sample.Core.csproj

```diff
@@ -8,7 +8,7 @@
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.*" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="2.1.*" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="2.1.*" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.1.*" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.*" />
```

Comment:
By removing this and NServiceBus references we'll have these transitive dependencies coming in anyways and it will not break.